src/webrtc/stats/summaryStats.ts

```diff
@@ -24,3 +24,6 @@ export interface TrackSummary {
     maxJitter: number;
     maxPacketLoss: number;
 }
+export interface AudioTrackSummary extends TrackSummary {
```

Comment:
I would suggest specialization is not needed. This specialization forces you to do an extra loop:
https://github.com/matrix-org/matrix-js-sdk/blob/fb3bdc39ecfdb12c0d6c3a3dd6e346d771517fde/src/webrtc/stats/trackStatsReporter.ts#L168
And if you not do an extra loop you would need at least an extra  `if - case`


And you initialize a default value anyway. 
https://github.com/matrix-org/matrix-js-sdk/blob/fb3bdc39ecfdb12c0d6c3a3dd6e346d771517fde/src/webrtc/stats/trackStatsReporter.ts#L143
For video this value could be simple 0 or -1